### PR TITLE
general update to all normal interactions files

### PR DESCRIPTION
**Pull request by Yzzuwyzzuf**
## About The Pull Request

added relationship events in every normal interactions file

## Why This Is Good For ClanGen

its always nice to have fresh stuff so why not go all the way with it and add to every file?

## Linked Issues

none

## Changelog/Credits

added relationship events to all normal interactions categories- fuzzy

### CODE REVIEW

File: resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json [hunk 1]
```diff
@@ -134,7 +155,8 @@
 			"m_c doesn't like the way that r_c seems to enjoy {PRONOUN/r_c/poss} influence over the Clan.",
 			"Much to m_c's dismay, r_c has asked {PRONOUN/m_c/object} to do another difficult task.",
 			"m_c thinks that r_c is casting a dark shadow over the whole Clan's reputation.",
-			"m_c is worried that r_c might be sending the Clan down a dark path."
+			"m_c is worried that r_c might be sending the Clan down a dark path.",
+			"m_c has been wondering if StarClan has actually been guiding r_c or the Dark Forest."
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json]:
```suggestion
			"m_c has been wondering if StarClan has actually been guiding r_c or if it's really the Dark Forest's influence."
```
Just making it clearer, the previous version was a bit too easy to misunderstand as "is starclan guiding r_c or is starclan guiding the dark forest"

File: resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json [hunk 2]
```diff
@@ -153,7 +175,9 @@
 			"m_c notices that r_c doesn't seem to care if {PRONOUN/r_c/poss} treatments hurt.",
 			"m_c overhears r_c complaining about giving herbs to cats that don't 'deserve' them.",
 			"m_c is startled at r_c's views on sharing herbs with other Clans.",
-			"m_c bristles after seeing r_c add more deathberries to {PRONOUN/r_c/poss} herb storage."
+			"m_c bristles after seeing r_c add more deathberries to {PRONOUN/r_c/poss} herb storage.",
+			"m_c stopped r_c before {PRONOUN/r_c/subject} gave the wrong herbs to another cat. The look in r_c's eyes as {PRONOUN/r_c/subject} did so scared {PRONOUN/m_c/object}...",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json]:
```suggestion
			"m_c stopped r_c before {PRONOUN/r_c/subject} gave the wrong herbs to another cat. The look in r_c's eyes as {PRONOUN/r_c/subject} did so scared m_c...",
```

making it clearer which cat is being referred to.  Generally, it's not a good idea to refer to two characters via pronouns one after the other.  A good rule of thumb is being sure to use a character's name before using their pronoun, this helps establish *who* the pronoun is referring to.

File: resources/dicts/relationship_events/normal_interactions/comfortable/increase.json [hunk 3]
```diff
@@ -25,7 +27,8 @@
 		"intensity": "low",
 		"interactions": [
 			"m_c is asking r_c to check {PRONOUN/m_c/object} for ticks.",
-			"m_c is happy to see r_c is the cat who is helping {PRONOUN/m_c/object} today."
+			"m_c is happy to see r_c is the cat who is helping {PRONOUN/m_c/object} today.",
+			"m_c didn't even have to ask r_c to check {PRONOUN/m_c/object} for ticks, {PRONOUN/r_c/subject} {VERB/r_c/were/was} on it the moment r_c started to scratch."
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/comfortable/increase.json]:
```suggestion
			"m_c didn't even have to ask r_c to check {PRONOUN/m_c/object} for ticks, r_c was on it the moment m_c started to scratch."
```

File: resources/dicts/relationship_events/normal_interactions/comfortable/increase.json [hunk 4]
```diff
@@ -39,7 +42,10 @@
 	{
 		"id": "comfort_inc_med1",
 		"interactions": [
-			"m_c helps r_c pick burrs out of {PRONOUN/r_c/poss} fur."
+			"m_c helps r_c pick burrs out of {PRONOUN/r_c/poss} fur.",
+			"m_c came across r_c napping and left them be, making sure no one else woke {PRONOUN/r_c/object} either.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/comfortable/increase.json]:
```suggestion
			"m_c came across r_c napping and left {PRONOUN/r_c/object} be, making sure no one else woke {PRONOUN/r_c/object} either.",
```

File: resources/dicts/relationship_events/normal_interactions/comfortable/increase.json [hunk 5]
```diff
@@ -83,7 +91,11 @@
 		"interactions": [
 			"r_c is listening to m_c's woes.",
 			"r_c is listening to m_c's troubles.",
-			"r_c noticed m_c was struggling, and offered to help {PRONOUN/m_c/object}."
+			"r_c noticed m_c was struggling, and offered to help {PRONOUN/m_c/object}.",
+			"r_c offered help to m_c during a hard time.",
+            "m_c offers to help with one of r_c's more tedious tasks.",
+			"m_c helps r_c with a tedious task and it makes the whole day a lot easier.",
+            "m_c had a nightmare and r_c offered to share {PRONOUN/r_c/poss} nest to comfort {PRONOUN/m_c/object}."
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/comfortable/increase.json]:
```suggestion
            "m_c had a nightmare and r_c offered to share {PRONOUN/r_c/poss} nest to comfort m_c."
```

File: resources/dicts/relationship_events/normal_interactions/comfortable/increase.json [hunk 6]
```diff
@@ -136,7 +149,9 @@
 		"id": "comfortable_kit_inc_med2",
 		"interactions": [
 			"m_c wants to snuggle with r_c.",
-			"m_c is hiding under a bush from r_c, but {PRONOUN/m_c/subject} can't stop giggling."
+			"m_c is hiding under a bush from r_c, but {PRONOUN/m_c/subject} can't stop giggling.",
+			"m_c wonders aloud if r_c would play with {PRONOUN/m_c/object} within earshot of r_c glancing over at {PRONOUN/m_c/object} expectantly.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/comfortable/increase.json]: This is rather unclear, see if you can reword it.  Perhaps something like: "m_c notices r_c glancing over at them expectantly. They pretend not to notice and instead wonder loudly if r_c would want to play with them."

File: resources/dicts/relationship_events/normal_interactions/dislike/decrease.json [hunk 7]
```diff
@@ -25,25 +27,43 @@
...
+            "m_c and r_c apologized over something petty they both did to each other."
 		]
 	},
 	{
 		"id": "dislike_de_med2",
 		"interactions": [
 			"m_c covered for r_c on something minor.",
-			"m_c cheers r_c up after a small disagreement."
+			"m_c cheers r_c up after a small disagreement.",
+			"m_c, while grabbing food for {PRONOUN/m_c/self}, saw m_c grabbing a really tasty looking rabbit and offered to share it with {PRONOUN/r_c/object}."
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/dislike/decrease.json]:
```suggestion
			"m_c, while grabbing food for {PRONOUN/m_c/self}, saw r_c grabbing a really tasty looking rabbit and asked to share it with {PRONOUN/r_c/object}."
```

File: resources/dicts/relationship_events/normal_interactions/jealousy/increase.json [hunk 8]
```diff
@@ -69,7 +105,9 @@
 	{
 		"id": "jealousy_kit_inc_med2",
 		"interactions": [
-			"m_c hisses at r_c for getting a bigger piece of prey."
+			"m_c hisses at r_c for getting a bigger piece of prey.",
+			"m_c is upset that r_c always wins in their games.",
+            "m_c whines and cries when r_c gets more attention then {PRONOUN/m_c/subject} get more attention."
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/jealousy/increase.json]:
```suggestion
            "m_c whines and cries when r_c gets more attention then {PRONOUN/m_c/subject}."
```

File: resources/dicts/relationship_events/normal_interactions/neutral.json [hunk 9]
```diff
@@ -83,24 +104,51 @@
 		"id": "neutral_interaction8",
 		"interactions": [
 			"r_c notices m_c sulking around camp again.",
+            		"r_c sits quietly by m_c.",
+            		"r_c offers to share tongues with m_c but {PRONOUN/r_c/subject} pass.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/neutral.json]:
```suggestion
            		"r_c offers to share tongues with m_c but {PRONOUN/m_c/subject} decline.",
```

File: resources/dicts/relationship_events/normal_interactions/platonic/decrease.json [hunk 10]
```diff
@@ -3,31 +3,57 @@
...
 		}
 	},
 	{
 		"id": "platonic_de_med1",
-		"intensity": "high",
+		"intensity": "medium",
 		"interactions": [
-			"m_c caught r_c complaining about {PRONOUN/m_c/object} behind {PRONOUN/m_c/poss} back."
+			"m_c caught r_c complaining about {PRONOUN/m_c/object} behind {PRONOUN/m_c/poss} back.",
+			"m_c interrupted r_c when {PRONOUN/r_c/subject} {VERB/r_c/was/were} very busy with an important task ruining {PRONOUN/r_c/poss} focus.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/platonic/decrease.json]:
```suggestion
			"m_c interrupted r_c when {PRONOUN/r_c/subject} {VERB/r_c/was/were} very busy with an important task, ruining {PRONOUN/r_c/poss} focus.",
```

File: resources/dicts/relationship_events/normal_interactions/platonic/decrease.json [hunk 11]
```diff
@@ -3,31 +3,57 @@
...
+            "m_c realized r_c was only pretending to pay attention to {PRONOUN/m_c/object} when {PRONOUN/m_c/subject} {VERB/m_c/was/were} talking.",
+            "m_c tried to talk to r_c but gave up seeing how clearly {PRONOUN/r_c/subject} wanted to be doing anything else.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/platonic/decrease.json]:
```suggestion
            "m_c tried to talk to r_c but gave up, seeing how clearly {PRONOUN/r_c/subject} wanted to be doing anything else.",
```

File: resources/dicts/relationship_events/normal_interactions/platonic/decrease.json [hunk 12]
```diff
@@ -3,31 +3,57 @@
...
+            "m_c overheard r_c talking about {PRONOUN/m_c/object} but when pressed for details, r_c denied mentioning {PRONOUN/m_c/object} at all.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/platonic/decrease.json]:
```suggestion
            "m_c overheard r_c talking about {PRONOUN/m_c/object}, but when pressed for details, r_c denied mentioning {PRONOUN/m_c/object} at all.",
```

File: resources/dicts/relationship_events/normal_interactions/platonic/decrease.json [hunk 13]
```diff
@@ -3,31 +3,57 @@
...
+            "m_c got up and walked away when m_c sat nearby.",
+            "m_c blatantly ignored r_c.",
+            "Lately, m_c's had some choice words for r_c with how {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} been acting lately...",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/platonic/decrease.json]:
```suggestion
            "Lately, m_c's had some choice words for r_c about how {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} been acting lately...",
```

File: resources/dicts/relationship_events/normal_interactions/platonic/decrease.json [hunk 14]
```diff
@@ -3,31 +3,57 @@
...
+            "m_c asked another cat to pass a message to r_c rather doing it {PRONOUN/m_c/self}.",
+            "m_c was passive aggressive towards r_c for taking something {PRONOUN/r_c/subject} knew {PRONOUN/m_c/subject} had {PRONOUN/m_c/poss} eye on all day.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/platonic/decrease.json]:
```suggestion
            "m_c was passive aggressive towards r_c for taking something {PRONOUN/r_c/subject} knew m_c had {PRONOUN/m_c/poss} eye on all day.",
```

File: resources/dicts/relationship_events/normal_interactions/platonic/decrease.json [hunk 15]
```diff
@@ -3,31 +3,57 @@
...
+            "r_c is caught tossing out a gift m_c JUST gave {PRONOUN/m_c/object}.",
```
Comment (scribblecrumb) [inline resources/dicts/relationship_events/normal_interactions/platonic/decrease.json]:
```suggestion
            "r_c is caught tossing out a gift m_c <i>just</i> gave {PRONOUN/m_c/object}.",
```

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
partial review before i take off the weekend ;)

File: resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json (hunk 1 above)
Comment (Yzzuwyzzuf) [reply resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json]: Giotcha, I see that now, thanks for the suggestion

File: resources/dicts/relationship_events/normal_interactions/comfortable/increase.json (hunk 6 above)
Comment (Yzzuwyzzuf) [reply resources/dicts/relationship_events/normal_interactions/comfortable/increase.json]: Hope this wording is better

**Pull request closed without merging**